python/google/protobuf/internal/message_test.py

```diff
@@ -2441,5 +2442,36 @@ def testSucceedOversizeProto(self):
     q.ParseFromString(self.p_serialized)
     self.assertEqual(self.p.field.payload, q.field.payload)
 
+
+@unittest.skipIf(api_implementation.Type() != 'cpp' or
+                 sys.version_info < (2, 7),
```

Comment:
Protobuf python only support python 2.7 and above, no need to do the filter here

Reply:
Ah…I copied this from https://github.com/protocolbuffers/protobuf/blob/d1e72c11e676ddb0aa852fbb3e3f968828a95381/python/google/protobuf/internal/message_test.py#L2385; should I also remove the explicit version check from the other test too?